torch/jit/_recursive.py

```diff
@@ -104,6 +104,8 @@ def infer_concrete_type_builder(nn_module, share_types=True):
         concrete_type_builder.set_module_list()
 
     class_annotations = getattr(nn_module, '__annotations__', {})
+    # Get user-annotated ignored attributes.
+    user_annotated_ignored_attributes = getattr(nn_module, "__jit_ignored_attributes__", set())
```

Comment:
`__torchscript_ignored__`? JIT is a little overloaded.

Reply:
:'( I just landed a change to name the equivalent class attribute for unused properties `__jit_unused_properties__`. I'm good with either, but I just want to make sure they're consistent. So I can either keep this or change both to `__torchscript_ignored_xxx__`.